.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,60 @@
+name: Bug Report
+about: Create a report to help us improve
+title: "bug: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the Bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to Reproduce
+      description: Tell us what actions you performed before the issue occurred
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behaviour
+      description: Tell us what should have happened?
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Behaviour
+      description: Tell us what happened instead
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: Copy the output of `asdf-info` here
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: plugins
+    attributes:
+      label: asdf plugins affected (if relevant)
+      description: The plugins you list here should appear in the list from `asdf info`
+  - type: textarea
+    id: additional-information
```

Comment:
Do we really need this field? It seems like the user could type everything in the description field.

Reply:
Not really, I was just copying over the existing templates